test/README.testsuite

```diff
@@ -52,7 +52,7 @@ directory (`OutputDirectory=`) to point to the other directory using `mkosi.loca
 After the image has been built, the integration tests can be run with:
 
 ```shell
-$ SYSTEMD_INTEGRATION_TESTS=1 meson test -C build --no-rebuild --suite integration-tests --num-processes "$(($(nproc) / 4))"
```

Comment:
this is called directly in debian/ubuntu so changing the name breaks that

Reply:
Hmm. This particular change I think makes the most sense. I hope we can do it somehow.